apps/parsley/src/context/LogContext/index.tsx

```diff
@@ -263,6 +268,10 @@ const LogContextProvider: React.FC<LogContextProviderProps> = ({
           setFilterLogic(v);
           Cookie.set(FILTER_LOGIC, v, { expires: 365 });
         },
+        setHighlightFilters: (v: boolean) => {
```

Comment:
This is consistent with the rest of the setters in the file, do you feel strongly about this?

Reply:
Sorry for the late reply. I don't think these are examples of shadowing since it isn't overwriting the previously declared value. This is creating a new object with those fields as params and is roughly equivalent to doing something like.
```ts
const preferences = {
toggleHighlightFilters: toggleHighlightFilters
}
```
I do agree with you however this is pretty complex and is likely better extracted into a separate hook/ reducer. I don't mind moving these out but I think it would be better suited to be in a separate ticket/hook since the refactor would get pretty complex/is pretty out of scope for this ticket. 
I filed a ticket to address it outside of the context of this pr. https://jira.mongodb.org/browse/DEVPROD-6967 What do you think?

Reply:
Hmm yeah I can see how that can be confusing DEVPROD-6967 should address that. We should be able to refactor these into dispatch calls after consolidating the settings reducers